x/fileurl.go

```diff
@@ -0,0 +1,110 @@
+package x
+
+import (
+	"net/url"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
+
+	"github.com/ory/x/logrusx"
+)
+
+// winPathRegex is a regex for [DRIVE-LETTER]:
+var winPathRegex = regexp.MustCompile("^[A-Za-z]:.*")
+
+// GetURLFilePath returns the path of a URL that is compatible with the runtime os filesystem
+func GetURLFilePath(u *url.URL) string {
+	if u == nil {
+		return ""
+	}
+
+	if u.Scheme != "file" && u.Scheme != "" {
```

Comment:
Let's invert this condition to make it more readable
```suggestion
	if u.Scheme == "file" || u.Scheme == "" {
```

Reply:
Hi Mr. De Morgan :wave:

Reply:
This is fixed